### PR TITLE
Visibility to map and map to visibility bag conversation tool basics, based on IDL codes

### PR DESCRIPTION
**Pull request by pgabor**
Something is not right. You can check it with:

```
import xrayvision
res, map_data = xrayvision.test_mapvis()
#res and map_data should be the same
#res is the visibility bag's obsvis values
#map_data is the generated map
```
After I get back, I try to figure out what is wrong, but in the mean time please check it too! Maybe I just overlooked something.

### CODE REVIEW

File: xrayvision/vis2map.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    ----------
+    | https://darts.isas.jaxa.jp/pub/ssw/gen/idl/image/vis/
+    |                   vis_spatial_frequency_weighting.pro
+    """
+    if type(spatial_frequency_weight) == int:
+        spatial_frequency_weight = np.ones((visin.shape[0],))
+    fov = float(bp_fov)
+    pixel = fov / 200.
+    npx = int(fov / pixel)
+    MAP = np.zeros((npx, npx))
```
Comment (dpshelio) [inline xrayvision/vis2map.py]: please, do not use upper case variables unless they are not globals in the file. Also, `map` is not a good name as it clashes already with python's `map`  and sunpy's `sunpy.map`.

**Comment by dpshelio** [comment]
The  code looks a bit too idl-ly...  (I'm so sorry for you, @pgabor. Hopefully you recover in the future 😛 )

**Comment by pgabor** [comment]
I will recover :D

**Comment by pgabor** [comment]
At least I hope so...

File: xrayvision/vis2map.py (hunk 1 above)
Comment (pgabor) [inline xrayvision/vis2map.py]: Thanks for the advice

**Pull request closed without merging**